packages/jest-phabricator/src/index.ts

```diff
@@ -3,23 +3,22 @@
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
- * @flow
  */
 
-import type {
+import {
   AggregatedResult,
   AggregatedResultWithoutCoverage,
   CoverageMap,
-} from 'types/TestResult';
+} from '@jest/types';
```

Comment:
This needs to be `import {TestResult} from '@jest/types'`, then do `TestResult. AggregatedResult` etc on usage sites

Reply:
Oh man! Thanks! :)